test/mithril.route.js

```diff
@@ -130,68 +129,7 @@ describe("m.route()", function () {
 		expect(spy).to.not.have.been.called
 	})
 
-	// FIXME: this causes others to fail
-	xdit("skips route change if subcomponent ctrl.onunload calls preventDefault", function (root) { // eslint-disable-line
```

Comment:
Please don't remove these tests. It would be best to try re-enabling them instead (assuming they pass).